src/multithreading.c

```diff
@@ -45,18 +45,9 @@ void wait_light_off( void )
 	pthread_mutex_unlock( &light_mutex );
 }
 
-int is_light_turned_on( bool ignore )
+int is_light_turned_on( void )
 {
-	int temp;
-
-	if (ignore)
-		return true;
-
-	pthread_mutex_lock( &light_mutex );
-	temp = run_light;
-	pthread_mutex_unlock( &light_mutex );
-
-	return temp;
+	return run_light;
```

Comment:
As this is only used in the beginning of the test, how much performance improvement we can get if we get ride of the mutex?

Reply:
This is also used within the send data path inner loops.